templates/starter/bicep/README.md

```diff
@@ -7,3 +7,65 @@ The following assets have been provided:
 - Infrastructure-as-code (IaC) files under the `infra` folder that demonstrate how to provision resources and setup resource tagging for `azd`.
 - A [dev container](https://containers.dev) configuration file under the `.devcontainer` directory that installs infrastructure tooling by default. This can be readily used to create cloud-hosted developer environments such as [GitHub Codespaces](https://aka.ms/codespaces).
 - Continuous deployment workflows for CI providers such as GitHub Actions under the `.github` directory, and Azure Pipelines under the `.azdo` directory that work for most use-cases.
+
+## Next Steps
+
+### Step 1: Add application code
+
+Initialize the service source code projects anywhere under the current directory. Ensure that all source code projects can be built successfully.
```

Comment:
Might be nice to number steps under each header for easier parsing.

Reply:
Let me know if you have specific improvements so I can incorporate. I haven't quite figured out how to massage the other sections to a consistent numbering format.

Reply:
I made some changes along these lines, it does read better